core/src/subgraph/runner.rs

```diff
@@ -425,7 +425,8 @@ where
                                     .unwrap()
                                     .remove(&self.inputs.deployment.id);
 
-                                error!(logger, "Subgraph failed with non-deterministic error: {}", e;
+                                let message = format!("{:#}", e).replace("\n", "\t");
+                                error!(logger, "Subgraph failed with non-deterministic error: {}", message;
```

Comment:
Here the message is formatted as in other places.

Reply:
Is that from using anyhow's `with_context`? That doesn't play nice with keeping things on one log line.

Reply:
The `{:#}` is to print anyhow causes, in a single line. Still newlines must be replaced because wasm traps included traces with newlines.